src/MiNET/MiNET/Net/MCPE Protocol.cs

```diff
@@ -6187,9 +6187,27 @@ protected override void ResetPacket()
 
 	public partial class McpeBossEvent : Packet<McpeBossEvent>
 	{
+		public enum BossEvent : uint
+		{
+			Add = 0,
+			PlayerAdded = 1,
+			Remove = 2,
+			PlayerRemoved = 3,
+			UpdatePercent = 4,
+			UpdateName = 5,
+			UpdateProperties = 6,
+			UpdateStyle = 7,
+		}
 
 		public long bossEntityId; // = null;
-		public uint eventType; // = null;
+		public BossEvent eventType = BossEvent.Add;
+		public string bossName = "";
+		public float bossHealthPercent = 1.0f; // from 0 to 1.0
+		public ushort darkenScreen = 12; // forgot what is it
```

Comment:
I'd be prepared to bet this is the same as PC, since it's an almost perfect copy pasta.

Reply:
Thanks. Check it today.

Reply:
Thanks. Check it out today.

Reply:
@dktapps I think you are right. But it changes nothing on the client side when I change it on the server.

Reply:
I checked it myself and there are some minor differences, but you're right about it being useless.